implementation/leprechaun/src/shared/test/helpers.ts

```diff
@@ -1,22 +1,66 @@
 import { Line } from '@app/line/database/line.entity';
+import { StopLineMapping } from '@app/line/database/stop-line-mapping.entity';
 import { Route } from '@app/route/database/route.entity';
+import { Stop } from '@app/stop/database/stop.entity';
 import { Vehicle } from '@app/vehicle/database/vehicle.entity';
+import { MikroORM } from '@mikro-orm/core';
+import { EntityManager } from '@mikro-orm/postgresql';
 
-export function createTimeOffsetFromNow(hours: number, seconds: number = 0): Date {
+export type TimeType = {
+  hours: number;
+  seconds?: number;
+};
+
+type Entity = {
+  name: string;
+};
+
+export type FuncOrPromise<T = unknown> = (() => Promise<T>) | Promise<T>;
+
+export function createTimeOffsetFromNow({ hours, seconds }: TimeType): Date {
+  if (seconds === undefined) {
+    seconds = 0;
+  }
   return new Date(new Date().valueOf() + (3600 * hours + seconds) * 1000);
 }
```

Comment:
```suggestion
export function createTimeOffsetFromNow({ hours, seconds = 0 }: TimeType): Date {
  return new Date(new Date().valueOf() + (3600 * hours + seconds) * 1000);
}
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_value

Reply:
Destructuring statements are still magic to me.